src/addNewcomment.js

```diff
@@ -0,0 +1,31 @@
+const addNewComment = () => {
+  const mealItem = document.querySelector('.meal-details');
+  const form = document.querySelector('.form');
+  const userName = form.elements.name;
+  const userComment = form.elements.comment;
+
+  const InvolvementApiKey = 'H1778Eipl0PRUFSJ8DNo';
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${InvolvementApiKey}/comments`
```

Comment:
- [ ] Good job so far 👍 Kindly display the name of the user who wrote the comment together with the comment written as shown in the original design 
- Your design:
![Annotation 2022-06-23 235107](https://user-images.githubusercontent.com/85113567/175428002-5bc56a4b-3f4c-47b3-8ace-15ac6d03e254.jpg)
- Required design:
![Annotation 2022-06-23 235309](https://user-images.githubusercontent.com/85113567/175428083-871b0cb8-38c4-4367-9223-cf700ba74e11.jpg)